docs/developer-guide/Server Methods/geotiff-imagery-directories.md

```diff
@@ -0,0 +1,38 @@
+# GeoTIFF Imagery Directories
```

Comment:
Can we rename the path to `docs/developer-guide/server-methods/serve-basemaps-with-geotiff-imagery-directories.md`
```suggestion
# Serve Basemaps with GeoTIFF Imagery Directories
```